changes.d/4983.fix.md

```diff
@@ -0,0 +1 @@
+Ensure the runahead limit is recomputed when legacy "suicide-triggers" are used to prevent erroneous stall in niche cases.
```

Comment:
```suggestion
Ensure the runahead limit is recomputed when legacy "suicide-triggers" are used, to prevent erroneous stall in niche cases.
```
(A bit ambiguous without the comma)